notebooks/nessie-iceberg-hive-demo-nba.ipynb

```diff
@@ -641,7 +641,7 @@
         }
       ],
       "source": [
-        "# We should see `allstar_games_stats` and the `new_total_stats` on the `etl` branch\n",
```

Comment:
note that this was the only time in this file where `new_total_stats` was mentioned (i guess copy paste error)